proof/crefine/ARM_HYP/Retype_C.thy

```diff
@@ -5360,7 +5360,7 @@ proof -
   let ?htdret = "(hrs_htd_update (ptr_retyp (vcpu_Ptr p)) (t_hrs_' (globals \<sigma>')))"
   let ?zeros = "from_bytes (replicate (size_of TYPE(vcpu_C)) 0) :: vcpu_C"
 
-  have "size_of TYPE(vcpu_C) = 456" (is "_ = ?vcpusz")
+  have "size_of TYPE(vcpu_C) = ?vcpusz"
```

Comment:
I'd add a comment that this is a sanity check, otherwise it's unused now and not declaring the var

Reply:
Good point, will add.